test/js/metrics-api.test.js

```diff
@@ -102,4 +106,46 @@ describe('Library for client-side usage analytics', () => {
       done()
     })
   })
+
+  it('appropriately determines whether to log to Sentry', () => {
+
+    console.log = jest.fn();
+
+    jest
+      .spyOn(SCPContextProvider, 'getSCPContext')
+      .mockReturnValue({environment: 'logged-env'})
```

Comment:
is there any significance to this `logged-env` string?  If not, let's replace it with either something that clearly indicates it's meaningless (e.g. 'something' or maybe 'anything-other-than-test'), or with either 'production' or 'development'.

Reply:
Yes, the significance is that it conveys the environment would be a logged one, without coupling it to implementation details.

This strikes me as a matter of taste; I'd love to see precedent in other code either way.  Your suggested change is fortunately trivial here, so I've gone ahead and implemented it.